src/panels/lovelace/hui-editor.ts

```diff
@@ -152,6 +155,9 @@ class LovelaceFullConfigEditor extends LitElement {
     } else if (!this._changed && window.onbeforeunload) {
       window.onbeforeunload = null;
     }
+    if (this._changed) {
+      this.requestUpdate();
```

Comment:
Why do we need the `requestUpdate`?

Reply:
When I tested this, that was the best I could come up with, that is now a few days ago, I can probably do something different now to handle it, I'll have a look after work.

Reply:
`_changed` is a property, so if that is updated it should re-render?

Reply:
Should, yes.
does, not so much 🤷‍♂ (can be tested here https://demo.home-assistant.io/#/lovelace/0)

I can probably find a better way later today.

Reply:
Found a better way, looking at styles now